java/libphonenumber/pom.xml

```diff
@@ -30,19 +30,26 @@
     </testResources>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
```

Comment:
The maven-jar-plugin changes are not related to the fix, but a cleanup.

org.apache.maven.plugins is still the groupId, it's an implicit default.